src/Aspire.Hosting.PostgreSQL/PostgresContainerImageTags.cs

```diff
@@ -11,4 +11,7 @@ internal static class PostgresContainerImageTags
     public const string PgAdminRegistry = "docker.io";
     public const string PgAdminImage = "dpage/pgadmin4";
     public const string PgAdminTag = "8.8";
+    public const string PgWebRegistry = "docker.io";
+    public const string PgWebImage = "sosedoff/pgweb";
+    public const string PgWebTag = "latest";
```

Comment:
We've been not using `latest` tags as much as possible due to non-determinism. See examples:

* https://github.com/dotnet/aspire/issues/4828
* https://github.com/dotnet/aspire/issues/5057

Reply:
now we use the `0.15.0` tag,Could you please import  `docker.io/sosedoff/pgweb:0.15.0`?

Reply:
I can do that.

Reply:
@eerhardt already did it!